packages/service/src/service/base.ts

```diff
@@ -33,8 +33,8 @@ export class BtcAssetsApiBase implements BaseApis {
     }
 
     const packedParams = params ? '?' + new URLSearchParams(pickBy(params, (val) => val !== undefined)).toString() : '';
-    const withOriginHeaders = this.origin ? { origin: this.origin } : void 0;
-    const withAuthHeaders = requireToken && this.token ? { Authorization: `Bearer ${this.token}` } : void 0;
+    const withOriginHeaders = this.origin ? { origin: this.origin } : undefined;
```

Comment:
```suggestion
    const withOriginHeaders = { origin: this.origin };
```

Reply:
If the origin is `undefined`, the origin field will not appear in the header and you can have a try.

Reply:
I split them for better readability, although it may have failed to achieve its job. Another solution is that these properties can also be packed into the fetch call:
```ts
fetch('xxx', {
  headers: {
    authorization: requireToken && this.token ? `Bearer ${this.token}` : undefined,
    origin: this.origin,
    ...headers,
  },
})
```

Reply:
LGTM